### DISCUSSION TITLE
How to wear cloth on model without changing any thing in cloth?

### DISCUSSION THREAD

**Discussion by naikbright** [lllyasviel/Fooocus#3802] [Q&A] [answered]
How to wear cloth on model without changing any thing in cloth?
cloths should be as is not change in design colour and pattern. exact cloth should be place on model provided in inpaint.

currently it placing cloth on inpaint model but change cloth design.

![image (8)](https://github.com/user-attachments/assets/c97e2eab-a6b1-4734-b410-373f11280372) --> Inpaint model 
![51P4jHXTERL _SX679_](https://github.com/user-attachments/assets/44374fcc-f96c-47e8-9581-92e33b92574b) --> cloth which needs to wear by inpaint model.
![image (12)](https://github.com/user-attachments/assets/9aa47529-fb94-4ff1-874b-bd0c38585772) --> output

**Comment by ghost** [comment, marked as answer]
Use a graphics software like photoshop or whatever. You can't do that in Fooocus alone.
